plugins/vendor/zoom/lib/model/kZoomEventHanlder.php

```diff
@@ -92,6 +107,21 @@ public function processEvent($event)
 		}
 	}
 	
+	protected function getEntryOwnerId($hostEmail)
```

Comment:
this logic does not take into consideration the different options that can be set for zoom user
why not using the id on the object returns on getEntryOwner on kZoomProcessor instead?